openedx/core/storage.py

```diff
@@ -44,6 +45,16 @@ class DevelopmentStorage(
     pass
 
 
+class S3ReportStorage(S3BotoStorage):
+    """
+    Storage for reports.
+    """
+    def __init__(self, acl=None, bucket=None, custom_domain=None, **settings):
```

Comment:
nit: Can you add the doc string to represent passing params here ? e.g. `acl` etc